templates/index.html

```diff
@@ -453,438 +494,435 @@ <h2>Do more with trusted open source</h2>
-                        with open source infrastructure
-                      </h3>
-                    </div>
-                    <div class="p-image--container u-hide--large">
-                      {{ image(url="https://assets.ubuntu.com/v1/1a25e055-1.png",
-                                            alt="",
-                                            width="2464",
-                                            height="1027",
-                                            hi_def=True,
-                                            attrs={
-                                            "class": "p-image--container__image",
-                                            },
-                                            loading="lazy") | safe
-                      }}
-                    </div>
-                    <div class="col-6 col-medium-6">
-                      <p>
-                        <i>“Canonical’s open source software sits at the heart of BT Group’s network cloud infrastructure, underpinning our 5G mobile core capability and providing the stability and flexibility required to support the increasing convergence of BT services with confidence.”</i>
-                      </p>
-                      <a class="p-cta-text u-hide--medium u-hide--small"
-                         href="https://ubuntu.com/engage/5G-with-open-source-infrastructure">Learn how BT unlocked scalability<i class="p-icon--arrow-right"></i></a>
-                    </div>
-                    <div class="col-3 col-medium-3">
-                      <p class="u-no-margin--bottom">
-                        <b>Greg McCall</b>
-                      </p>
-                      <p class="u-text--muted">
-                        Chief Networks Officer,
-                        <br />
-                        BT Group
-                      </p>
-                    </div>
-                    <a class="p-cta-text u-hide--large"
-                       href="https://ubuntu.com/engage/5G-with-open-source-infrastructure">Learn how BT unlocked scalability<i class="p-icon--arrow-right"></i></a>
-                  </div>
+      </div>
+    </div>
+    <div tabindex="0"
+          role="tabpanel"
+          id="scale-confidence-tab"
+          class="p-tabs__panel"
+          aria-hidden="false"
+          aria-labelledby="scale-confidence">
+      <div class="p-section">
+        <div class="row">
+          <div class="col-3" style="padding-top: 0.45rem;">
+            {{ image(url="https://assets.ubuntu.com/v1/3de63226-BT%20group.png",
+                            alt="BT Group",
+                            width="150",
+                            height="57",
+                            hi_def=True,
+                            loading="lazy") | safe
+            }}
+          </div>
+          <div class="col-9 col-medium-6">
+            <div class="p-section">
+              <div class="row">
+                <div class="col-9 col-medium-6">
```

Comment:
Maybe not in scope since this is on prod and not related to js-vs not, but the extra row and col divs on lines 564-565 aren't needed since this is already within a row and defined cols on line 562